### ISSUE TITLE
acpid: handler.sh exports unnecessary variables

### ISSUE THREAD

**Issue by sbiberhofer** [voidlinux/void-packages#3299]
The handler.sh script shipped by default contains variables that are neither used later in the script, nor work correctly once multiple users are logged in. They also depend on dbus, which may or may not be present at all (also, the check in line 19 is a noop because of line 16). 

The commit looks like it is a hack to permit notifications to be send to X sessions and while this might be nice for some people, it should most definitely not be part of the default configuration and probably be removed from the default handler.sh. 

https://github.com/voidlinux/void-packages/blob/cad36257bf8238e0b3547c02ff52700eb28f9a93/srcpkgs/acpid/files/handler.sh#L13#L22

**Comment by chneukirchen** [comment]
I agree, that code is silly and if anything, belongs into a custom ZZZ script.

**Comment by thypon** [comment]
https://github.com/voidlinux/void-packages/pull/2486
